DataLink.tex

```diff
@@ -257,6 +255,13 @@ \subsubsection{Recursive DataLink}
 DataLink). This service link would be described with both a service type
 (as in \ref{sec:useStandard}) and content type.
 
+\subsubsection{Datasets linked to an astronomical source}
+
+There are  a lot of catalogs of astronomical sources made available using VO services such as ConeSearch \citep{2008ivoa.specQ0222P} or TAP services. For some catalogs “associated data” are available. These data include images from which sources have been extracted, or imaging the object in  case of extended objects, as well as additional observations such as Spectra or Time Series of the source and even spectral cubes and Time Series of images for extended or varying objects. The \blinks  response obtained for the source id can allow to easilly retrieve all these associated data in one shot.
+
+\subsubsection{Metadata and data related to provenance entities}
+
+The IVOA Provenance datamodel \citep{pr:provdm} represents metadata tracing  the history of data. This information can be retrieved through ProvTAP \citep{iwd:provtap} or ProvSAP \citep{iwd:provsap}  DAL services.  The Entity instances represent  the state of the data items between various steps of the data processing flow. “Entities” can be hooked to lists of real datasets retrieval and previews, or Obscore, SimDM and SSA metadata describing them  using the \blinks endpoint.
```

Comment:
I am not sure how this PR relates to #10, since it requests the same change here. To make sure that this doesn't get lost, copying my comment from there:

This is true only for `DatasetEntity` objects in Provenance. However there are also `ValueEntity` objects (f.e. to configuration parameters, as in the MuseWISE implementation) which cannot be hooked to lists of real datasets retrieval and previews. IMO this should be limited to `DatasetEntity` instances.
```suggestion
The IVOA Provenance datamodel \citep{pr:provdm} represents metadata tracing  the history of data. This information can be retrieved through ProvTAP \citep{iwd:provtap} or ProvSAP \citep{iwd:provsap}  DAL services.  The Entity instances represent  the state of the data items between various steps of the data processing flow. “DatasetEntities” can be hooked to lists of real datasets retrieval and previews, or Obscore, SimDM and SSA metadata describing them  using the \blinks endpoint.
```

Reply:
The best way to avoid conflict with other standards is to not restate things from that spec with too much detail. 

However, in this case I cannot tell if this advice is aimed at (i) linking from a dataset to one or more services that provide the provenance metadata or (ii) using datalink in a provenance service response to link to datasets. Sounds more like the latter, in which case either accept Ole's change or simplify the text a little to be less specific.